packages/core/src/components/viewer-grid/viewer-grid.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  h,
+  Component,
+  Prop,
+  State,
+  Host,
+  Element,
+  Event,
+  EventEmitter,
+} from "@stencil/core";
+import * as focusTrap from "focus-trap";
+
+type PanelSize = "small" | "medium" | "large";
+
+@Component({
+  tag: "dso-viewer-grid",
+  styleUrl: "viewer-grid.scss",
+  shadow: true,
+})
+export class ViewerGrid {
+  @Prop()
+  panelOpen = false;
+
+  @State()
+  panelSize: PanelSize = "large";
+
+  @Event()
+  closeAside!: EventEmitter<MouseEvent | KeyboardEvent>;
+
+  @Element()
+  host!: HTMLElement;
+
+  overlay!: HTMLDivElement;
+
+  overlaySlot!: HTMLDivElement;
```

Comment:
Ik denk dat het niet verstandig is om `overlay` en `overlaySlot` van een non-null assertion operator te voorzien. Het is fraaier om het component vergevingsgezind met missende slots te laten omgaan.

Reply:
Done.